### DISCUSSION TITLE
How do I compile the latest verison on MacOS?

### DISCUSSION THREAD

**Discussion by twiggs462** [ihhub/fheroes2#8308] [Q&A] [answered]
I have an older version that I found online. It is 1.0.6. It looks like 1.0.11 is the latest. When I go into a battle - it never goes into the battle it just says that I won or lost... it never actually does anything. I assume this is a bug that may have been fixed, but I do not know how to properly compile it.

Any help?
<img width="686" alt="Screenshot 2024-01-19 at 7 40 12 PM" src="https://github.com/ihhub/fheroes2/assets/42215828/db3c3f54-7bb4-419f-85eb-3d982429a310">

**Comment by zenseii** [comment]
Hi, @jderosa3!

> When I go into a battle - it never goes into the battle it just says that I won or lost... it never actually does anything.

This is the fheroes2-specific auto-resolve feature for battles. It can be turned off in the adventure map settings, found by pressing the button with the computer icon found on the panel on the right side. It is the bottom right option.

Alternatively you can leave it on and press the button that says RESTART and it will let you play the battle manually.

As for your questions about compiling we have a guide on how to install it for Mac users.

**Reply by zenseii** [reply, marked as answer]
Here is the installation guide:
https://github.com/ihhub/fheroes2/blob/master/docs/INSTALL.md#macos

If you still prefer to compile it yourself then here's the compilation guide:
https://github.com/ihhub/fheroes2/blob/master/docs/DEVELOPMENT.md

**Comment by twiggs462** [comment]
Thanks for the quick reply. I'll check the guide again. Appreciate the feedback on the battle setting. Thank you.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
